hedera-node/hedera-app/src/main/java/com/hedera/node/app/Hedera.java

```diff
@@ -250,14 +255,16 @@ public Hedera(
                         """,
                 HEDERA);
         bootstrapConfigProvider = new BootstrapConfigProviderImpl();
-        bootstrapConfig = bootstrapConfigProvider.getConfiguration();
+        final Configuration bootstrapConfig = bootstrapConfigProvider.getConfiguration();
         version = getNodeStartupVersion(bootstrapConfig);
         servicesRegistry = registryFactory.create(constructableRegistry, bootstrapConfig);
+        registerPlatformStateSchema();
```

Comment:
Why is the platform service registered differently from all other services? Why is it an anonymous inner class in `Hedera` rather than a regular class similar to other services?

Reply:
>Why is the platform service registered differently from all other services? 

It has to be registered first. Other services may need access to platform state accessor for their purposes. 

>Why is it an anonymous inner class in Hedera rather than a regular class similar to other services?

It doesn't have to be anonymous but it's doing nothing but tying the schema and the service registry together. So, I thought it's not pulling its weight as a full-blown class.

Reply:
> Other services may need access to platform state accessor for their purposes.

Do other services *currently* need access to platform state?

Reply:
It looks like it's no longer relevant with Michael's change.